Bottom: 83ad039686851a0476ac87deb96fa9264608ca1b
Top:    5d1d741a57632e678266a3ec1e25e6e03813bbe9
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-02-10 12:23:51 +0000

Avoid clearing pins / vias from polygons... it is already done!








---

diff --git a/src/draw.c b/src/draw.c
index 60e1175..9420bb1 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -1314,6 +1314,7 @@ ClearPin (PinTypePtr Pin, int Type, int unused)
       AddPart (Pin);
       return;
     }
+#if 0
   /* Clear the area around the pin */
   if (TEST_FLAG (SQUAREFLAG, Pin))
     {
@@ -1336,6 +1337,7 @@ ClearPin (PinTypePtr Pin, int Type, int unused)
     {
       gui->fill_circle (Output.pmGC, Pin->X, Pin->Y, half);
     }
+#endif
   if ((!TEST_FLAG (PINFLAG, Pin) && !PCB->ViaOn)
       || (TEST_FLAG (PINFLAG, Pin) && !PCB->PinOn))
     return;
